.github/workflows/Windows.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Setup vcpkg
         uses: lukka/run-vcpkg@v11
         with:
-          vcpkgGitCommitId: 501db0f17ef6df184fcdbfbe0f87cde2313b6ab1
+          vcpkgGitCommitId: 9edb1b8e590cc086563301d735cae4b6e732d2d2
```

Comment:
Well I have thought about this, I think in general DuckDB should be leading in what commit is pinned for extensions through the `.github/workflows/_extension_distribution.yml`. Extensions can then override this which is useful for example right now, the azure sdk needed updating and this means i can still deploy binaries compatible with 0.9.1 even though i use a more recent vcpkg commit. 

The fact that the vcpkg commit is even necessary here stems from another problem that needs fixing: Currently these workflows still exist because they cannot really reuse the built extensions from the `.github/workflows/_extension_distribution.yml` workflow. Ideally extensions can write workflows that download the binaries produced by `.github/workflows/_extension_distribution.yml`, then setup some testing environment, then run their tests with the pre-built binary. However this would require some glue work and possibly some refactoring to do this cleanly. 

I think any other direction is probably not really worth it tbh, but happy to discuss tomorrow

Reply:
Sure! It was more of a generic comment, I also think it's OK here.